azure-pipelines-integration-lsp.yml

```diff
@@ -21,7 +21,7 @@ pr:
 jobs:
 - job: VS_Integration_LSP
   pool:
-    name: NetCorePublic-Pool
+    name: NetCore1ESPool-Svc-Public
     queue: $(queueName)
```

Comment:
You should use demands: ImageOverride -equals here

Reply:
ah, I didn't save that file before committing. thanks for the catch!

Reply:
actually -- does anyone know where this variable comes from? because it will need to be changed as well. I can't find any references to this file in YAML so I'm assuming it's a build-time variable?

Reply:
`$queueName` is a pipeline variable. For integration tests we need to occasionally move them to or from the preview scouting queue depending on whether we've taken new VS dependencies or not.

Reply:
I expect a similar change would need to be made to the azure-pipelines-integration.yml

Reply:
Thank you @JoeRobich. We'll have to change those pipeline variables to the new queue names. Can you link me to the build in question?

Reply:
Roslyn integration: https://dev.azure.com/dnceng/public/_build?definitionId=245
Roslyn LSP integration: https://dev.azure.com/dnceng/public/_build?definitionId=930
Roslyn Scouting integration: https://dev.azure.com/dnceng/public/_build?definitionId=913

Reply:
@JoeRobich Because we'll be using different pools for servicing and R&D now, using a pipeline variable isn't tenable so I've removed the reliance on it with the latest push. Please check the updated description for an image of the new UX.

Reply:
@jonfortescue I am not sure this is a good solution for us. To update the queue as a parameter would require merging a code change and flowing it to the necessary branches. The goal is to be able to control the queue without the time and effort to do that. 

Also, our scouting queue runs on a nightly schedule, are you able to set a parameter as part of the schedule configuration?

Reply:
@JoeRobich Updating the queue parameter does not require a code change, you can do it at queue time via the UX.

Reply:
Integration tests aren't typically queued manually

Reply:
Ah, I understand! Sorry for the miscommunication. Let me investigate further.

Reply:
Per conversation offline, switched to hardcoding the queues for servicing.